update_ip/ip_getters/base.py

```diff
@@ -1,42 +1,49 @@
 import re
-import urllib2
 import socket
+import urllib2
 
 socket.setdefaulttimeout(5)
 
+
 class GetIpFailed(Exception):
     pass
 
-ip_regex= re.compile("\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}")
+
+# Check for valid IP Addresses
+ip_regex = re.compile(r"(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)")
```

Comment:
Yeah there are some good modules for validate IP, but while updating the program this can be a workaround, at least it doesn't allow IPs like 999.999.999.999 and can avoid parse invalid IPs from html.